client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/tasks.tsx

```diff
@@ -127,6 +133,7 @@ export const launchpadFlowTasks: LaunchpadFlowTaskList = {
 	free: [
 		'setup_free',
 		'design_selected',
+		'domain_upsell',
 		'first_post_published',
 		'design_edited',
```

Comment:
I think we should add this conditional task to show up in all launchpad flows when on a free plan/ no custom domain, not just the free flow.

Its fair to follow up with other plans, but we should at least start with free, build, and write.